### PR TITLE
Apply image template and add pip

### PR DESCRIPTION
**Pull request by joao-p-marques**
Some final testing a polishing of https://github.com/Tecnativa/image-template

@Yajo what do you think?

@Tecnativa
TT27819

### CODE REVIEW

**Review by yajo — APPROVED** [review]

**Comment by yajo** [comment]
The build took more than 6h and expired.

Please rebase on top of #4 and update the template after fixing https://github.com/Tecnativa/image-template/pull/9. Remember that #4 includes removal of one arch.

All that should make builds finish in ~45 minutes without cache.

**Comment by joao-p-marques** [comment]
Changes made @Yajo 
WDYT?

**Comment by yajo** [comment]
This is pure YAML verboseness poetry 😻 

I don't know how the world can still live without copier.

Well, just merge https://github.com/Tecnativa/image-template/pull/10, apply here and 👌🏼

File: Dockerfile [hunk 1]
```diff
@@ -3,6 +3,8 @@ ENV LANG=C.UTF-8 \
     PIPX_BIN_DIR="/usr/local/bin" \
     PIPX_HOME=/usr/local/share/pipx
 RUN dnf install -y \
+        python \
+        python-pip \
```
Comment (yajo) [inline Dockerfile]: Please respect alphabetical order

**Pull request merged**